working-group-lifecycle.md

```diff
@@ -0,0 +1,67 @@
+# Working Group Life Cycle
+
+Contributors interested in starting a new working group are encouraged to first review the existing
+working groups and projects. It is very likely that an existing working group could be extended to
+cover the new topic. In that case please bring your time and energy into that existing working
+group. Your contributions are very welcome!
+
+However, if no existing effort covers the topic feel free to make use of the existing working group
+meetings, mail lists, and/or slack channels to solicit support for a new working group.
+
+Please note that running a working group successfully requires an extended effort.
+Be prepared to commit at least several hours per week for a year or more.
+
+The Working Group Life Cycle begins with establishing commitments from other interested
+contributors then making a proposal to the Technical Advisory Committee (TAC). If accepted, the
+Working Group will begin as `Incubating`. This is a boot strapping phase for the group to adopt
+OpenSSF community policies and norms. Once those are met the group becomes `Active` and will be
+fully recognized in OpenSSF. Either upon completing its objectives or no longer making forward
+progress a project will move to `End-of-Life`. The details for each phase follow.
+
+## To become `Incubating`
+
+* Proposal of scope for review by TAC
+    * This is to help ensure limited overlap with existing WG
+* Have met at least 5 times
+    * For these, meeting notes (or ideally recordings) are public
+* Have at least 5 interested individuals from at least 3 different organizations attending regularly
+* 1 TAC sponsor
+    * TAC sponsor agrees to attend WG meetings regularly
+    * TAC sponsor does not need to have a formal role in WG, e.g., chair
+    * TAC sponsor requests TAC approval 
+* TAC will vote to approve or provide constructive guidance
+
+## Once `Incubating`:
+
+* Operate as part of the OpenSSF adhering to community policies
+* Deliver quarterly reviews to TAC
+* Complete and request TAC approval of [readme.md](https://github.com/ossf/project-template/blob/main/README.md)
+* Meet on a regular cadence. Meetings are public, recorded, and on the calendar
+* Have access to community resources (Zooms, YouTube channels, GitHub, Slack channels, etc.)
+* Can request funding/other resources (subject to TAC/GB approval)
+    * NOTE: _At the time of this draft, funding and resources beyond collaboration tools have not been established in the OpenSSF. Working groups should expect that their main resource is the community contributions they are able to recruit._
+
+## To become `Active`:
+
+* Have met at least 20 times
+* Have at least 10 regular members
+* Request TAC approval. TAC will vote to approve or provide constructive guidance
+
+## Once `Active`
+
+* (same as incubating, plus)
+* Have at least annual goals and metrics for success
```

Comment:
This one still doesn't make sense to me as a working group lead. What does a WG get in return for the extra overhead of producing annual goals and metrics?

I'm guessing this would get dropped over time and not strictly enforced, which is maybe not a bad thing.

Reply:
I think I originally suggested this because we wanted WGs to set goals and have a vision of what they're working on. But yeah, I agree with you that this is not necessarily the right thing to aim for - I also don't think there are necessarily expectations around the goals being met.
Maybe there's nothing 'additional' to do once you're active, as long as you keep doing the above. I'd be OK to remove.